include/mongoose.hrl

```diff
@@ -31,6 +31,13 @@
 
 -define(MONGOOSE_URI, <<"https://www.erlang-solutions.com/products/mongooseim.html">>).
 
+-define(APPLY_SAFELY(F),
```

Comment:
maybe not mongoose.hrl, could be apply_sefely.hrl...

Also, it is not APPLY_SAFELY(F) anymore, it is `catch` :D  So, maybe some better name.

Reply:
I could happily create such header and simply use the header from both source and test and kill the file.

When it comes to naming, oh naming is hard and it has always been called `safely` so, dunno :D

Reply:
because it was doing erlang:apply(F). Now it does F.
Maybe better, idk xdd:
`-define(SAFELY(Code), try Code....`

Reply:
Done 😃